errors/invalid-images-config.md

```diff
@@ -18,7 +18,7 @@ module.exports = {
     // limit of 50 domains values
     domains: [],
     path: '/_next/image',
-    // loader can be 'default', 'imgix', 'cloudinary', or 'akamai'
+    // loader can be 'default', 'imgix', 'cloudinary', 'imageengine' or 'akamai'
```

Comment:
```suggestion
    // loader can be 'default', 'imgix', 'cloudinary', 'imageengine', or 'akamai'
```

Reply:
Ah, and I just got used to removing the last comma :)
https://getitwriteonline.com/articles/oxford-comma/

I'll update this, thanks!